utils/docker/Dockerfile.centos.7

```diff
@@ -12,13 +12,69 @@ LABEL maintainer="daos@daos.groups.io"
 # Intermittent cache-bust.  Used to reduce load on the actual CACHEBUST later.
 ARG CB0
 
+# Switch to using DNF instead of yum.
+#
+# If a local distro repository is supplied, then we should use it
+# and disable using the distro repositories.
+# The Docker image starts out with the DISTRO provided GPG keys
+# on the disk, but not installed.  These must be isntalled before
```

Comment:
typo

Reply:
The coverity build stage is always run with this dockerfile set to quickbuild as true, because we only want the DAOS code to be scans.